ci/test_cudf_polars_polars_tests.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+set -eou pipefail
+
+# We will only fail these tests if the PR touches code in pylibcudf
+# or cudf_polars itself.
+# Note, the three dots mean we are doing diff between the merge-base
+# of upstream and HEAD. So this is asking, "does _this branch_ touch
+# files in cudf_polars/pylibcudf", rather than "are there changes
+# between upstream and this branch which touch cudf_polars/pylibcudf"
+# TODO: is the target branch exposed anywhere in an environment variable?
+if [ -n "$(git diff --name-only origin/branch-24.08...HEAD -- python/cudf_polars/ python/cudf/cudf/_lib/pylibcudf/)" ];
+then
+    HAS_CHANGES=1
+else
+    HAS_CHANGES=0
+fi
+
+RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
+RAPIDS_PY_WHEEL_NAME="cudf_polars_${RAPIDS_PY_CUDA_SUFFIX}" RAPIDS_PY_WHEEL_PURE="1" rapids-download-wheels-from-s3 ./dist
+
+# Download the cudf built in the previous step
+RAPIDS_PY_WHEEL_NAME="cudf_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 ./local-cudf-dep
+python -m pip install ./local-cudf-dep/cudf*.whl
+
+rapids-logger "Install cudf_polars"
+python -m pip install $(echo ./dist/cudf_polars*.whl)
+
+rapids-logger "Clone polars"
+git clone https://github.com/pola-rs/polars.git
+TAG=$(python -c 'import polars; print(f"py-{polars.__version__}")')
+pushd polars
+git checkout ${TAG}
+# Install requirements for running polars tests
+rapids-logger "Install polars test requirements"
+python -m pip install -r py-polars/requirements-dev.txt -r py-polars/requirements-ci.txt
+popd
```

Comment:
```suggestion
TAG=$(python -c 'import polars; print(f"py-{polars.__version__}")')
git clone https://github.com/pola-rs/polars.git --branch ${TAG} --depth 1
# Install requirements for running polars tests
rapids-logger "Install polars test requirements"
python -m pip install -r polars/py-polars/requirements-dev.txt -r polars/py-polars/requirements-ci.txt
```

Reply:
Might as well do a shallow clone, and no need to change dirs.

Reply:
Good idea, done.